deploy/dg_operator.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: infinispan-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: infinispan-operator
+  template:
+    metadata:
+      labels:
+        name: infinispan-operator
+    spec:
+      serviceAccountName: infinispan-operator
+      containers:
+        - name: infinispan-operator
+          image: docker.io/rigazilla/rigazilla-operator:i0.03
```

Comment:
this points to my repo to ease quick testing. should be changed when an official operator image with this pr will be available